pages/about.tsx

```diff
@@ -0,0 +1,47 @@
+const AboutPage = () => {
+  return (
+    <div className="bg-black z-20 text-white max-w-full fixed inset-0 overflow-y-auto overflow-x-auto">
+      <nav className="flex justify-between ml-8 mr-8 mt-3 h-12">
+        <a href="/">Atlas of Ownership</a>
+        <a href="/" className="text-lg">X</a>
+      </nav>
```

Comment:
we'll eventually want to move this nav bar out i suspect and check if next has an internal link element, but just hardcoded here for starters